### PR TITLE
make: Clean up coverage files

### PR DESCRIPTION
**Pull request by luise**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,8 +41,7 @@ check: gocheck jscheck
 
 clean:
 	govendor clean -x +local
-	rm -f *.cov.coverprofile cloud/*.cov.coverprofile minion/*.cov.coverprofile
-	rm -f *.cov.html cloud/*.cov.html minion/*.cov.html
+	find . -name "*.cov*" -type f -delete
```
Comment (kklin) [inline Makefile]: The second `*` is a bit too greedy, it'd be safer to be explicit about the extensions. E.g. "./node_modules/cliui/.coveralls.yml" was matched when I ran that find command.

Btw `find . -name '*.cov' -or -name '*.cov.html'` is how I would have multiple matches.

Comment (luise) [reply Makefile]: Oh, nice! I'll update!

Comment (kklin) [reply Makefile]: Oh, I tried it locally, and I think you need to group the match for the `-delete` to work?

```
find . -name '*.cov' -or -name '*.cov.html' -type f
```

Comment (luise) [reply Makefile]: Oh, I see. I just tested it without the `-delete`, and that found all the files. Thanks! Updated!

**Pull request merged**